auth_saml/README.rst

```diff
@@ -0,0 +1,180 @@
+====================
+SAML2 Authentication
+====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--auth-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-auth/tree/15.0/auth_saml
```

Comment:
Change everything all the related from 15.0 to 16.0

Reply:
Not as necessary, because README is autogenerated, but always a good idea.

The way to do it is running the following command:

`oca-gen-addon-readme --repo-name=server-auth --branch=16.0 --addon-dir=auth_saml`

([Source](https://mail.google.com/mail/u/0/h/1pq68r75kzvdr/?v%3Dlui)).

Reply:
I updated the changelog readme file and regenerated the readme.

Reply:
@moylop260 README has been updated as requested.

Could you review again, please?